config/reports.json

```diff
@@ -240,5 +282,19 @@
 			"bytesVideo",
 			"reqVideo"
 		]
+	},
+	"crux": {
+		"name": "Chrome User Experience Report",
+		"summary": "Insights from the RUM data in the Chrome UX report.",
```

Comment:
Loading performance as experienced by real-world Chrome users across a diverse set of real-world hardware and network conditions. The provided data is powered by real user measurement (RUM) of key loading user experience metrics across the public web, provided by the [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report/).

Reply:
Done